### PR TITLE
Tech Used Added

### PR DESCRIPTION
**Pull request by krniya**
Tech used added

### CODE REVIEW

**Comment by reapedjuggler** [comment]
Can you also mention Stargate it would be helpful

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ An open source data gateway.
 
 **Discord bot using Stargate**
 
+**Technology Used:**
```
Comment (eddiejaoude) [inline README.md:10]: Suggestion using a heading

```suggestion
## Technology Used
```

Comment (krniya) [reply README.md]: Made the requested changes.

**Review by Vyvy-vi — COMMENTED** [review]
LGTM-!
Could we also link to stargate?

**Comment by eddiejaoude** [comment]
PR #24 already merged with this information. Thank you for raising 👍

**Pull request closed without merging**